reports/dasharo-hcl-report

```diff
@@ -175,34 +173,39 @@ cat /sys/class/input/input*/id/bustype > logs/input_bustypes.log
 update_result "Input bus types"  logs/ioports.err.log
 printf '################################                                   |\r'
 
-# echo "Trying to read firmware image with flashrom..."
-# Some regions may be not available so we need to use specific regions to read
-check_intel_regions
-if [ $BOARD_HAS_FD_REGION -eq 1 ]; then
-  # Use safe defaults. Descriptor may contain additional regions not detected
-  # by flashrom and will return failure when attempted to be read. BIOS and
-  # Flash descriptor regions should always be readable. If not, then we have
-  # some ugly case, hard to deal with.
-  FLASHROM_ADD_OPT_READ="--ifd -i fd -i bios"
-  if [ $BOARD_HAS_ME_REGION -eq 1 ] && [ $BOARD_ME_REGION_LOCKED -eq 0 ]; then
-    # ME region is not locked, read it as well
-    FLASHROM_ADD_OPT_READ+=" -i me"
-  fi
-  if [ $BOARD_HAS_GBE_REGION -eq 1 ] && [ $BOARD_GBE_REGION_LOCKED -eq 0 ]; then
-    # GBE region is present and not locked, read it as well
-    FLASHROM_ADD_OPT_READ+=" -i gbe"
-  fi
+# flashrom does not support QEMU. TODO: this could be handled in a better way:
+if [ "${SYSTEM_VENDOR}" != "QEMU" ] && [ "${SYSTEM_VENDOR}" != "Emulation" ]; then
+  check_flash_chip
+
+  # echo "Trying to read firmware image with flashrom..."
+  # Some regions may be not available so we need to use specific regions to read
+  if [ $BOARD_HAS_FD_REGION -eq 1 ]; then
+    # Use safe defaults. Descriptor may contain additional regions not detected
+    # by flashrom and will return failure when attempted to be read. BIOS and
+    # Flash descriptor regions should always be readable. If not, then we have
+    # some ugly case, hard to deal with.
+    FLASHROM_ADD_OPT_READ="--ifd -i fd -i bios"
+    check_intel_regions
```

Comment:
Doesn't this function set `BOARD_HAS_FD_REGION`? In that case it has to be outside this `if`

Reply:
Fixed [here](https://github.com/Dasharo/dts-scripts/compare/ed6c6089caa0e541f15150b15a79a7e26c6ae365..9aa00a04166f712f638acc875e4004c28ece5c32)